app/models/user.rb

```diff
@@ -21,4 +23,5 @@ def has_company?(id)
   def find_supplier_admin(id)
     SupplierAdmin.find_by(user_id: id)
   end
```

Comment:
If we have time, we should go back through and scope this search through the users that belong to this SupplierAdmin only.